pipelines/internal/commands/run/run.go

```diff
@@ -324,6 +325,10 @@ func buildRequest(filename, project string) (*genomics.RunPipelineRequest, error
 		action := gsutil("cp", "/google/logs/output", *output)
 		action.Flags = []string{"ALWAYS_RUN"}
 		delocalizers = append(delocalizers, action)
+		if *logsInterval != 0 {
```

Comment:
This isn't a localizer (it's really a type of delocalizer, but we don't want to run it in that batch).

It should move down to just below where the actions slice is created (and be added first before anything else)

Reply:
Done.